Code/Framework/AzToolsFramework/Tests/Prefab/PrefabInstanceDomGeneratorTestFixture.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Prefab/PrefabInstanceDomGeneratorTestFixture.h>
+#include <Prefab/PrefabTestDomUtils.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+
+namespace UnitTest
+{
+    void PrefabInstanceDomGeneratorTestFixture::SetUpEditorFixtureImpl()
+    {
+        PrefabTestFixture::SetUpEditorFixtureImpl();
+
+        m_instanceDomGeneratorInterface = AZ::Interface<InstanceDomGeneratorInterface>::Get();
+        EXPECT_TRUE(m_instanceDomGeneratorInterface);
+
+        m_prefabFocusPublicInterface = AZ::Interface<PrefabFocusPublicInterface>::Get();
+        ASSERT_TRUE(m_prefabFocusPublicInterface);
+
+        SetUpInstanceHierarchy();
+    }
+
+    void PrefabInstanceDomGeneratorTestFixture::SetUpInstanceHierarchy()
+    {
+        // Level        <-- override WorldX of Tire and WorldX of Wheel container
+        // | Car        <-- override WorldX of Tire
+        //   | Wheel
+        //     | Tire
+
+        const AZStd::string carPrefabName = "CarPrefab";
+        const AZStd::string wheelPrefabName = "WheelPrefab";
+        const AZStd::string tireEntityName = "Tire";
+
+        AZ::IO::Path engineRootPath;
+        m_settingsRegistryInterface->Get(engineRootPath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
+        AZ::IO::Path carPrefabFilepath(engineRootPath);
+        carPrefabFilepath.Append(carPrefabName);
+        AZ::IO::Path wheelPrefabFilepath(engineRootPath);
+        wheelPrefabFilepath.Append(wheelPrefabName);
```

Comment:
Looks like it can be consolidated as mentioned by @cgalvan in the other PR:
```cpp
AZ::IO::Path carPrefabFilepath = engineRootPath / carPrefabName;
AZ::IO::Path wheelPrefabFilepath = engineRootPath / wheelPrefabName;
```